src/app/tests/suites/certification/PICS.yaml

```diff
@@ -378,46 +384,6 @@ PICS:
     - label: "Does the device implement the MaxCoolSetpointLimit attribute"
       id: A_MAXCOOLSETPOINTLIMIT
 
-    - label:
-          "Does the device implement the ControlSequenceOfOperation attribute"
-      id: A_CONTROLSEQUENCEOFOPERATION
-
-    - label:
-          "Does the device implement receiving the SetpointRaiseLower command?"
-      id: CR_SetpointRaiseLower
-
-    - label:
-          "Does the device implement the UnoccupiedCoolingSetpoint attribute?"
-      id: A_UNOCCUPIEDCOOLINGSETPOINT
-
-    - label:
-          "Does the device implement the UnoccupiedHeatingSetpoint attribute?"
-      id: A_UNOCCUPIEDHEATINGSETPOINT
-
-    - label: "Does the device implement the MinSetpointDeadBand attribute?"
-      id: A_MINSETPOINTDEADBAND
-
-    - label: "Does the device implement the Tolerance attribute?"
-      id: A_TEMPERATURE_TOLERANCE
```

Comment:
From CI:
```

"A_TEMPERATURE_TOLERANCE" is not a known PICS item in src/app/tests/suites/certification/PICS.yaml
"TSTAT_AUTO" is not a known PICS item in src/app/tests/suites/certification/PICS.yaml
"TSTAT_COOL" is not a known PICS item in src/app/tests/suites/certification/PICS.yaml
"TSTAT_HEAT" is not a known PICS item in src/app/tests/suites/certification/PICS.yaml
"TSTAT_OCC" is not a known PICS item in src/app/tests/suites/certification/PICS.yaml
"TSTAT_SB" is not a known PICS item in src/app/tests/suites/certification/PICS.yaml
"TSTAT_SCH" is not a known PICS item in src/app/tests/suites/certification/PICS.yaml
"TSTST-S.F02" is not a known PICS item in src/app/tests/suites/certification/PICS.yaml
```

Reply:
fixed the test case and ci pics values to remove these